finalPrep/modZeroHero.js

```diff
@@ -55,30 +87,61 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - citiesDestroyed (array)
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
-
+var scaryMonster = {
+  name: 'Bigfoot',
+  smell: 'rancid meat',
+  weight: 1500,
+  citiesDestroyed: ['Detroit', 'Chicago', 'Aurora'],
+  luckyNumbers: [1, 3, 5, 9],
+  address: '1234 Cryptid Lane, Florida, 32004'
+}
 
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
+//   - energyLevel = 50
 
 // - Create the following class methods
 //   - sayName, should print the hero's name to the console
 //   - maximizeEnergy, should update the energyLevel to 1000
 //   - gainPower, should take an argument of a number and INCREASE the powerLevel by that number
 
 // - Create 2 instances of your SuperHero class
-
+class SuperHero {
+  constructor(name, superpower, age){
+    this.name = name
+    this.superpower = superpower,
+    this.age = age
+    this.archNemesis = 'The Syntax Error'
+    this.powerLevel = 100
+    this.energyLevel = 50
+  }
+  sayName() {
+    console.log(this.name);
+  }
+  maximizeEnergy() {
+    this.energyLevel = 1000;
+  }
+  gainPower() {
```

Comment:
According to the instructions, `gainPower` should accept an argument of a number and increase the `powerLevel` instance variable by that number.